spec/acceptance/support-skeletons/additional_storage/Vagrantfile

```diff
@@ -5,6 +5,7 @@
 
 Vagrant.configure("2") do |config|
   config.vm.box = "infernix/tinycore"
+  config.vm.box_check_update = false
```

Comment:
Meant to ask as well, how does this help with stability? Or is it just handy to disable when running a lot?

Generally I'm thinking that the CI here should be using the latest images rather than those that it was run with the first time. There should be caching in place to avoid this costing a lot of time by default, bit of it's useful to disable in certain cases I could see using an env var instead being better.

Reply:
I was curious if this was intentional. Maybe I can switch to using an environment variable to toggle? Either way you hit the nail on the head - it was painful when running the test suite dozens of times in a day.

I'll put some thinking into this one.